can_tools/scrapers/official/AL/al_vaccine.py

```diff
@@ -34,3 +34,133 @@ def normalize(self, data):
         locations_to_drop = [0, 99999]
         data = data.query("location != @locations_to_drop")
         return data
+
+
+class ALCountyVaccineSex(ALCountyVaccine):
+    variables = {
+        "F": CMU(
+            category="total_vaccine_doses_administered",
+            measurement="cumulative",
+            unit="doses",
+            sex="female",
+        ),
+        "M": CMU(
+            category="total_vaccine_doses_administered",
+            measurement="cumulative",
+            unit="doses",
+            sex="male",
+        ),
+        "U": CMU(
+            category="total_vaccine_doses_administered",
+            measurement="cumulative",
+            unit="doses",
+            sex="unknown",
+        ),
+    }
+
+    variable_columns = ["RECIP_SEX"]
+    sheet_num = 6
+
+    def fetch(self):
+        service = "Vaccination_Dashboard_AGOL_v4_PUBLIC_VIEW"
```

Comment:
looks like we can make this a class attribute also, then we wouldn't need to define a `fetch` method in the subclasses

Reply:
done